x/foundation/README.md

```diff
@@ -0,0 +1,1619 @@
+* [Parameters](#parameters)
+* [State](#state)
+* [Msg Service](#msg-service)
+    * [Msg/UpdateDecisionPolicy](#msgupdatedecisionpolicy)
+    * [Msg/UpdateMembers](#msgupdatemembers)
+    * [Msg/LeaveFoundation](#msgleavefoundation)
+    * [Msg/SubmitProposal](#msgsubmitproposal)
+    * [Msg/WithdrawProposal](#msgwithdrawproposal)
+    * [Msg/Vote](#msgvote)
+    * [Msg/Exec](#msgexec)
+    * [Msg/UpdateCensorship](#msgupdatecensorship)
+    * [Msg/Grant](#msggrant)
+    * [Msg/Revoke](#msgrevoke)
+    * [Msg/FundTreasury](#msgfundtreasury)
+    * [Msg/WithdrawFromTreasury](#msgwithdrawfromtreasury)
+* [Events](#events)
+    * [EventUpdateDecisionPolicy](#eventupdatedecisionpolicy)
+    * [EventUpdateMembers](#eventupdatedmembers)
+    * [EventLeaveFoundation](#eventleavefoundation)
+    * [EventSubmitProposal](#eventsubmitproposal)
+    * [EventWithdrawProposal](#eventwithdrawproposal)
+    * [EventVote](#eventvote)
+    * [EventExec](#eventexec)
+    * [EventUpdateCensorship](#eventupdatecensorship)
+    * [EventGrant](#eventgrant)
+    * [EventRevoke](#eventrevoke)
+    * [EventFundTreasury](#eventfundedtreasury)
+    * [EventWithdrawFromTreasury](#eventwithdrawedfromtreasury)
+* [Client](#client)
+    * [CLI](#cli)
+    * [gRPC](#grpc)
+
+# Concepts
+
+## Authority
+
+`x/foundation`'s authority is a module account associated with the foundation
+and a decision policy. It is an "administrator" which has the ability to add,
+remove and update members in the foundation.
+`x/foundation` has several messages which cannot be triggered but by the
+authority. It includes membership management messages, and other messages which
+controls the assets of the foundation.
+
+**Note:** The authority is a module account, which means no one has the private
+key of the authority. Hence, foundation members MUST propose, vote and execute
+the corresponding proposal.
+
+## Decision Policy
+
+A decision policy is the rules that dictate whether a proposal should pass or
+not based on its tally outcome.
+
+All decision policies generally would have a mininum execution period and a
+maximum voting window. The minimum execution period is the minimum amount of
+time that must pass after submission in order for a proposal to potentially be
+executed, and it may be set to 0. The maximum voting window is the maximum time
+after submission that a proposal may be voted on before it is tallied.
+
+The chain developer also defines an app-wide maximum execution period, which is
+the maximum amount of time after a proposal's voting period end where the
+members are allowed to execute a proposal.
+
+The current foundation module comes shipped with two decision policies:
+threshold and percentage. Any chain developer can extend upon these two, by
+creating custom decision policies, as long as they adhere to the
+`DecisionPolicy` interface:
+
++++ https://github.com/Finschia/finschia-sdk/blob/392277a33519d289154e8da27f05f9a6788ab076/x/foundation/foundation.go#L90-L103
```

Comment:
Please update all old links.

Reply:
I will update the links after this PR merged, because it would be squashed.